taniumthreatresponse_connector.py

```diff
@@ -464,25 +475,35 @@ def _handle_list_connections(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
 
         ret_val, response = self._list_connections(action_result)
-
         if phantom.is_fail(ret_val):
             self.save_progress('List connections failed')
             return action_result.get_status()
 
         summary = action_result.update_summary({})
         try:
-            for status in [x['status'] for x in response if 'status' in x]:
-                summary['{}_connections'.format(
-                    CONNECTION_STATUS[status])] = len([x for x in response if 'status' in x and x['status'] == status])
-            for r in response:
-                action_result.add_data(r)
-                summary['total_connections'] = len(response)
+            active_connections = 0
+            inactive_connections = 0
+            for resp in response:
+                if 'status' in resp and resp['status'] == 'disconnected':
```

Comment:
Could also do `resp.get('status')`?

Reply:
Instead of directly getting status value now using .get() method.